hazelcast-spring/src/test/java/com/hazelcast/spring/CustomSpringJUnit4ClassRunner.java

```diff
@@ -28,6 +30,8 @@
 
 public class CustomSpringJUnit4ClassRunner extends SpringJUnit4ClassRunner {
 
+    private static final ThreadLocal<String> TEST_NAME_THREAD_LOCAL = new InheritableThreadLocal<String>();
```

Comment:
This thread local is not needed. `TestLoggingUtils.setThreadLocalTestMethodName()` sets needed parameter.

Reply:
you are right, i will delete it 👍